app/controllers/api/v1/accounts/pictures_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Api::V1::Accounts::PicturesController < Api::BaseController
+  before_action -> { doorkeeper_authorize! :write, :'write:accounts' }
+  before_action :require_user!
+  before_action :set_picture
+  before_action :validate_picture_param
+
+  def destroy
+    @account = current_account
+    UpdateAccountService.new.call(@account, { @picture => nil, "#{@picture}_remote_url" => '' }, raise_error: true)
```

Comment:
The `#{@picture}_remote_url` bit should not be relevant for local users, which the endpoint is for. I'd rather remove it.

```suggestion
    UpdateAccountService.new.call(@account, { @picture => nil }, raise_error: true)
```